runtimes/datascience/ubi8-python-3.8/utils/bootstrapper.py

```diff
@@ -46,7 +46,8 @@
 F = TypeVar("F", bound="FileOpBase")
 
 logger = logging.getLogger("elyra")
-enable_pipeline_info = os.getenv("ELYRA_ENABLE_PIPELINE_INFO", "true").lower() == "true"
+enable_pipeline_info = os.getenv(
```

Comment:
Why did we split these lines here? I thought they were readable as they were.
The same for (85, 87, 110, 116, 188, 239, 262, 324, 380, ... and many others!) 
I would suggests to keep as much similarity as we can with this one https://github.com/opendatahub-io/elyra/blob/dspv2/elyra/kfp/bootstrapper.py

Reply:
This is due to the auto-linter.
I will fix it.